### DISCUSSION TITLE
Installation Tutorial

### DISCUSSION THREAD

**Discussion by mattwells** [vwt12eh8/hassio-ecoflow#50] [Show and tell]
I have created a video demonstraiting how to setup Hassio Ecoflow in Home Assistant, my first tutorial so I would appreciate any feedback. 

https://youtu.be/KFpb1SVWdIA

Thanks,
Matt

**Comment by Maxctran** [comment]
used it and it worked for my delta pros and extra batteries thanks so much, are you able to show a video how to set it up with the Smart Home Panel? I can see on other discussions it has been possible but I am not technical enough to understand how to do it.

**Reply by mattwells** [reply]
Oh that is fantastic, what version of the Delta Pro firmware are you using? (it will tell you in the Ecoflow app)

Sadly I don't have a Smart Home Panel so unfortunatly I won't be able to cover that.

**Reply by Maxctran** [reply]
v1.0.1.18 of the Delta Pro firmware which it says is the newest as there are no updates. I think reading the posts online it wasn't working then EcoFlow changed their minds and opened up the ports and it seems to be working again. Thats a shame that you don't have a Smart Home Panel, its been a pain in the backside because of the lack of integration with existing solar but I am trying to essentially get it all to automatically charge with my existing household AC solar on my roof via Home Assistant. I have to manually change charge rate which has wasted countless hours since its been installed at Christmas last year. 

I have worked out that Glow's In Home Display Smart Meter can integrate with Home Assistant as well but I am due to have my Smart Meter upgraded next week from 1st gen SMETS1 to newer SMETS2 so will order it then and hope I am work out how to get them both to work together to change the charge rate depending on how much solar electricity I am generating.
